src/chromeDebugAdapter.ts

```diff
@@ -222,7 +225,7 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         // Disconnect before killing Chrome, because running "taskkill" when it's paused sometimes doesn't kill it
         super.disconnect(args);
 
-        if (this._chromeProc && !hadTerminated) {
+        if ( (this._chromeProc || (!this._chromeProc && this._chromePID)) && !hadTerminated) {
```

Comment:
Shouldn't happen with the current code, but since this can go through when this._chromeProc is null, add a null check below where we call .kill on it.

Reply:
Good catch. 

Done.